exercises/sum-of-multiples/canonical-data.json

```
@@ -136,6 +136,15 @@
         "limit": 10000
       },
       "expected": 39614537
+    },
+    {
+      "description": "the factor 0 does not affect the sum of multiples of other factors",
+      "property": "sum",
+      "input": {
+        "factors": [3,0],
```

```suggestion
        "factors": [3, 0],
```

apparently GitHub lets me make a Suggestion now. I wonder how this works. Might as well use it to add a space after the comma, for consistency